insert_dummy.py

```diff
@@ -0,0 +1,126 @@
+import datetime
```

Comment:
umm. can we not touch the database outside the server code? Let's do this with `requests`?

Reply:
Ok sure. Sorry for that. I'll change it.

Reply:
But then at this point of time, without a database call I can only call the questions post method. There is the ContestProblems Model which gets populated in the server.py program which takes in question as an input. So when I run the server program, it will fail.
Am I going about this right or is there another way to it ?

Reply:
you mean there is no "contest" for you to create a "contestproblem" against right? If that is the case then you're missing an api somewhere. If we insert the questions via requests it essentially simulates what the browser would have to do. Perhaps this can be the flow:

1. add quesions without attaching them to contests
2. create a contest by selecting from a list of questions